.travis.yml

```diff
@@ -4,8 +4,7 @@ sudo: required
 language: go
 
 go:
-  - 1.8.x
-  - tip
+  - 1.10.x
```

Comment:
I think docker 17.06 ee is currently built with Go 1.12.x, so it may be good to have that version in this list at least (but given that Go 1.12.x will reach EOL soon, probably have 1.13.x in there as well).

Any reason to remove `tip`?

Reply:
OK, I will add go 1.12, 1.13. `tip` also caused golint error.